Predicting antimicrobial resistance genes
=========================================

In this section, we will describe how you can:

* `Short guide for those in a hurry: what is the predicted antimicrobial resistance profile of my isolate?`_
* `Predicting antimicrobial resistance genes in your isolate`_

Short guide for those in a hurry: what is the predicted antimicrobial resistance profile of my isolate?
-------------------------------------------------------------------------------------------------------

A quick way to find out whether your isolate is predicted to display antimicrobial resistance is to look at the predicted antimicrobial resistance  for your isolate in Vibriowatch.

Once you have sequencing reads or a genome assembly for your isolate to Vibriowatch (see `Is my isolate Vibrio cholerae? <https://vibriowatch.readthedocs.io/en/latest/assemblies.html#short-guide-for-those-in-a-hurry-is-my-isolate-vibrio-cholerae>`_), Vibriowatch will display a piechart showing the species of your isolates are, e.g.:

.. image:: Picture7.png
  :width: 650

To view the report pages for your isolates, which will tell you their predicted antimicrobial resistance, you need
to click on the 'View genomes' link in the middle of the piechart.
  
This will bring up a list of your isolates in Vibriowatch, looking something like this:

.. image:: Picture8.png
  :width: 650
  
To go to the report page for a particular isolate, click on the link on the left in the 'Name' column, e.g. '1_S1_L001'.

If you scroll down the 'report page' for your isolate, you will find a section with the heading 'AMR - Antimicrobial resistance'. 

For example, here is 'Antimicrobial resistance' part of the report page for an isolate HCUF-01, which was collected in Haiti in 2010 and
published by `Hasan et al 2012`_:

.. _Hasan et al 2012: https://pubmed.ncbi.nlm.nih.gov/22711841/

.. image:: Picture95.png
  :width: 650
  
We can see that HCUF-01 has resistance genes *strA* and *strB*, predicted to give resistance to aminoglycosides such as streptomycin;
resistance gene *sul2*, predicted to give resistance to sulfonamides (e.g. sulfisoxazole); 
resistance genes *catB9* and *floR*, predicted to give resistance
to chloramphenicol; resistance gene *varG*, predicted to give resistance to beta lactam class antibiotics including carbapenems, cephalosporins,
and ampicillin; and resistance gene *dfrA1*, predicted to give resistance to trimethoprim.

HCUF-01 also has some variants predicted to give antimicrobial resistance: variants *nfsA* (R169C) and *nfsB* (Q5STOP), predicted to give resistance
to nitrofurans (e.g. nitrofurantoin, furazolidone); and variants *gyrA* (S83I) and *parC* (S85L), predicted to give resistance to quinolones (e.g. ciprofloxacin, nalidixic acid). 
  
If you found this useful, you may want to read through the rest of the tutorial to find out more details.
  
Predicting antimicrobial resistance genes in your isolate
---------------------------------------------------------

Bla bla

Contact
-------

I will be grateful if you will send me (Avril Coghlan) corrections or suggestions for improvements to my email address alc@sanger.ac.uk

