Vibriowatch
===========

**Vibriowatch** is a database of high quality genomes and metadata for *Vibrio cholerae*, with robust bioinformatics capability for virulence, antimicrobial resistance, and finding related isolates. Vibriowatch contains `>4000` published *V. cholerae* genomes, and manually-curated metadata from the literature, including strain names; place, date and source of isolation; and phenotypic data for biotype, serogroup, serotype and antimicrobial resistance. Vibriowatch is aimed for use by clinicians and public health workers studying cholera outbreaks, as well as for research biologists investigating *V. cholerae* genomic variation and evolution.

First, a brief overview of this tutorial: we'll introduce the *Vibrio cholerae* reference genome, and explain what other genomic data is available for *V. cholerae*. We'll introduce `PathogenWatch`_ and its *V. cholerae* component Vibriowatch. We'll then explain how you can upload your own reads or genome to Vibriowatch, and analyses that you can carry out using Vibriowatch on your genome, as well as the genomes that we have already uploaded into Vibriowatch. 

.. _PathogenWatch: https://pathogen.watch/

That is, in this tutorial, we will describe how users can use Vibriowatch to:

* upload reads or assembled genomes.
* verifying that their isolate's species is *V. cholerae*.
* view the statistics for their assembly.
* carry out MLST analysis of their isolate.
* compare their isolate to Vibriowatch's *V. cholerae* reference genomes.
* build a phylogenetic tree to check if their isolate belongs to the pandemic lineage.
* use cgMLST to find closely related isolates.
* build a phylogenetic tree of their isolate and its closest relatives.
* predict virulence genes.
* predict antimicrobial resistance genes.
* predict plasmids.

Acknowledgements
----------------

I would like here to acknowledge the great help and work of my colleagues at the Wellcome Sanger Institute, especially Dr Matthew Dorman, Dr Florent Lassalle, Dr Sina Beier, Dr Alyce Taylor-Brown, Dr Adrian Cazares, Sam Dougan, and Prof. Nicholas Thomson, and all of the Thomson group.

Thank you also to our fantastic collaborators Dr Corin Yeats and Prof. David Aanensen who have developed Pathogenwatch at the Centre for Genomic Pathogen Surveillance, and have extensively adapted it for Vibriowatch.

Thank you to Dr Josefina Campos (INEI-ANLIS Malbran, Argentina),
Dr Neelam Taneja and Nisha Singh (PGIMER Chandigarh, India), 
and Dr Yann Boucher (National University of Singapore),
who have advised us on Vibriowatch.

Lastly, but very importantly, we would
like to say thank you to our funders who have funded our work. These are the Bill and Melinda Gates Foundation, and also the University of Oxford, Wellcome Trust, and Wellcome Sanger Institute. 

Contact
-------

I will be grateful if you will send me (Avril Coghlan) corrections or suggestions for improvements to my email address alc@sanger.ac.uk

Contents
--------

.. toctree::

   vibriogenomes
   assemblies
   mlst
   closerelatives
   virulenceAMR
   


