*Vibrio cholerae* genomic data
==============================

In this section, we will describe:

* `The Vibrio cholerae reference genome`_.
* `What genomic data is available for Vibrio cholerae`_.
* `Pathogenwatch and Vibriowatch`_.

The Vibrio cholerae reference genome
------------------------------------

The *Vibrio cholerae* genome was first sequenced by `Heidelberg et al 2000`_. They sequenced strain N16961, which belongs to the current pandemic lineage (7PET lineage). They found that the genome is four Megabase and consists of two circular chromosomes, a larger chromosome of about three Megabase and a smaller chromosome of about 1.1 Megabase. Together, these two chromosomes have about 4000 genes, and their GC content of the chromosomes is about 47%. 

.. _Heidelberg et al 2000: https://pubmed.ncbi.nlm.nih.gov/10952301/

.. image:: Picture17.png
  :width: 300

It's worth bearing in mind that N16961 is the reference strain, but there may be a wide variety of genomic variation in the *V. cholerae population* worldwide. Some interesting examples have already been reported. `Johnson et al 2015`_ reported a *V. cholerae* isolate that has the two chromosomes fused into one big chromosome. `Okada et al 2015`_ reported an isolate with a third replicon of nearly one Megabase so that its genome size was five Megabase. As we sequence more and more genomes of *V. cholorae* isolates, we are likely to see more and more genomic variation.

.. _Johnson et al 2015: https://pubmed.ncbi.nlm.nih.gov/25977434/

.. _Okada et al 2015: https://pubmed.ncbi.nlm.nih.gov/26079534/

What genomic data is available for Vibrio cholerae
--------------------------------------------------

Since the year 2000, genome sequencing technologies have greatly improved, and this has meant that many different *V. cholerae* isolates have had their genomes sequenced. An example of this with a key paper by `Chun et al 2009`_, who sequenced the genomes of 23 diverse *V. cholerae* isolates that had been collected over nearly 100 years. These isolates included both clinical and environmental isolates from many different countries. 

.. _Chun et al 2009: https://pubmed.ncbi.nlm.nih.gov/19720995/

Since then, many more *V. cholerae* genomes have been sequenced. This plot shows with the number of isolates sequenced in 22 key papers ranging from the year 2000 at the top to 2022 at the bottom:

.. image:: Picture19.png
  :width: 500

Together these 22 key papers have published 3254 *V. cholerae* genomes, of which 84% belonged to the current pandemic lineage (7PET lineage). All of these genome data has been useful to clinicians and epidemiologists to try and understand the spread of *V. cholerae* around the world and also to scientists and epidemiologists interested in *V. cholerae* genome evolution, for example, the spread of antibiotic resistance genes, and the presence or absence of virulence genes. 

Of course, these 22 key papers do not include all papers that have published *V. cholerae* genomes. In fact, if you look at the `NCBI database`_ or `ENA database`_, you will be able to find about 5700 *V. cholerae* genome assemblies. These will include some genomes that were published in papers that I have not listed here, but also some genomes that have not yet been published. Furthermore, if you look in the ENA database, you will see about 14,500  *V. cholerae* sequence runs, which are unassembled reads for *V. cholerae* isolates. This reflects the fact that many researchers in the past have not assembled reads into assemblies but simply aligned the reads to the reference genome. 

Clearly, there is a huge amount of genomic data available for *V. cholerae*. Our goal in the VibrioWatch project is to make all this data available and useful for clinicians, epidemiologists and researchers studying *V. cholerae*.

.._NCBI database: https://www.ncbi.nlm.nih.gov/data-hub/genome/?taxon=666

.._ENA database: https://www.ebi.ac.uk/ena/browser/advanced-search

Pathogenwatch and Vibriowatch
-----------------------------

Contact
-------

I will be grateful if you will send me (Avril Coghlan) corrections or suggestions for improvements to my email address alc@sanger.ac.uk


